XRTK-Core/Packages/com.xrtk.core/Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -779,7 +779,7 @@ public GameObject GetTrackedAreaVisualization()
                 layer = DefaultIgnoreRaycastLayer
             };
 
-            var position = MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform.position;
+            var position = MixedRealityToolkit.CameraSystem.MainCameraRig.PlayspaceTransform.position;
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`